crates/cairoc/README.md

```diff
@@ -0,0 +1,4 @@
+# Hieratika Cairoc
+
+A wrapper to generate `FlatLowered` representation of Cairo code and convert it into the `flo`
+format for ingestion by Hieratika.
```

Comment:
```suggestion
```suggestion
A wrapper to generate a `FlatLowered` representation of Cairo code and convert it into the `flo` format to enable separate compilation and linking.
```